sdk/python/pulumi_gcp/monitoring/get_uptime_check_ips.py

```diff
@@ -38,7 +58,7 @@ def __await__(self):
             uptime_check_ips=self.uptime_check_ips)
 
 
-def get_uptime_check_i_ps(opts=None):
+def get_uptime_check_ips(                         opts: Optional[pulumi.InvokeOptions] = None) -> AwaitableGetUptimeCheckIPsResult:
```

Comment:
@komalali caught this `get_uptime_check_i_ps` => `get_uptime_check_ips ` breaking change. We'll need to address this in the codegen before we can release.

Reply:
https://github.com/pulumi/pulumi/pull/5202

Reply:
We'll need to update the `go.mod` require for pulumi/pulumi in this PR to reference this commit: https://github.com/pulumi/pulumi/commit/62925433421370e32e614f2d2647f577673148bc